__tests__/commands/_helpers.js

```diff
@@ -131,3 +131,11 @@ export async function run<T, R>(
     throw new Error(`${err && err.stack} \nConsole output:\n ${out}`);
   }
 }
+export const setPlatform = (platform : string) : string => {
+  const previous = process.platform;
+  Object.defineProperty(process, 'platform', {
```

Comment:
I'm going to redo that. It's not effective at all 😰 I originally was pondering just passing platform into **run** so the test could directly send it, but, to your point, that seems corny.